tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -83,9 +84,11 @@ public function userSearchesUsingWebDavAPI(
 			= "<?xml version='1.0' encoding='utf-8' ?>\n" .
 			"	<oc:search-files xmlns:a='DAV:' xmlns:oc='http://owncloud.org/ns' >\n" .
 			"		<oc:search>\n";
-		if ($scope !== null) {
-			$scope = \trim($scope, "/");
-			$resourceID = $this->featureContext->spacesContext->getResourceId($user, $spaceName ?? "Personal", $scope);
+		if ($spaceName !== null) {
+			$resourceID = $this->featureContext->spacesContext->getResourceId($user, $spaceName, $scope ?? "");
+			$pattern .= " scope:$resourceID";
+		} elseif ($scope !== null) {
+			$resourceID = $this->featureContext->spacesContext->getResourceId($user, "Personal", $scope);
 			$pattern .= " scope:$resourceID";
 		}
 		$body .= "<oc:pattern>$pattern</oc:pattern>\n";
```

Comment:
IMHO, doing so would probably make the `$resourceID` undefined in snippet: `$pattern .= " scope:$resourceID";` since `$resourceID` is defined only inside those `if` and `elseif` part but `$pattern` is defined above also.